src/tasks/mod.rs

```diff
@@ -4,3 +4,4 @@ pub mod exec;
 pub mod info;
 pub mod upload;
 pub mod download;
+pub mod module;
```

Comment:
Please add a newline at the end of file :)

Reply:
done